docker-compose.yml

```diff
@@ -95,3 +95,21 @@ services:
     image: redpencil/varnish-post:1.0.0
     environment:
       BACKEND_HOST: database
+  publisher:
+    image: lblod/reglement-publish-service:0.0.1
+    environment: 
+      NODE_ENV: "development"
+    links:
+      - database:database
+    volumes:
+      - /Users/oscar.rodriguez/dev/redpencil/reglement-publish-service:/app/
```

Comment:
you cant put absolute paths to your computer in the main docker-compose file and expect them to work